src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py

```diff
@@ -419,13 +419,22 @@ def list_app_service_plans(resource_group_name=None):
     return plans
 
 
+def _linux_sku_check(sku):
+    tier = _get_sku_name(sku)
+    if tier == 'BASIC' or tier == 'STANDARD':
+        return
+    format_string = '{0} is not a valid sku for linux, please use one of the following: {1}'
```

Comment:
Suggest `usage error: {0} is not a valid sku for linux plan, please use one of the following....`